### DISCUSSION TITLE
Does `pipenv install --dev cheddar` install as a dev dependency?

### DISCUSSION THREAD

**Discussion by JDLH** [pypa/pipenv#5135] [Q&A]
If I have a developer tool `cheddar`, a Python package, and I want to install it as a development rather than a runtime dependency, do I use the option `--dev` to accomplish that?  In other words, is 

```pipenv install --dev cheddar```

the correct way to install dev packages?  

I ask because the documentation does not seem to say that.  The [**pipenv install** documentation](https://pipenv.pypa.io/en/latest/basics/#pipenv-install) says, 
> `--dev` — Install both `develop` and `default` packages from Pipfile.

I am guessing that correctly describes the behaviour of `pipenv install --dev` without a package name. 

But from my tests, if I include a package name, then the package gets installed and listed in the `[dev-packages]` section of `Pipfile`. That is, `pipenv install --dev cheddar` does not "Install both `develop` and `default` packages from Pipfile."

If this use of `--dev` is correct, then I think it would be clearer for that line to say, 

> `--dev` — with `package names`, install those packages as development dependencies. With no names, install both development and default dependencies listed in the Pipfile.

(Also, I don't see where the terms "default packages" and "develop packages" are defined. The advanced topics mention them as "dependencies", which I think is a better term. I think "default dependencies" are required to run the app, and are listed in the `[packages]` section of the Pipfile; while "development dependencies" are require to develop and build the app but not to run it, so do not need to be distributed to end users of the app. Maybe "runtime" is better than "default" here?)

(Also, the **pipenv install** documentation does not seem to say that `pipenv install` with no package names and no `--dev` option will install the default dependencies from the Pipfile.)

**Comment by kalebmckale** [comment]
As you likely discovered, when specifying packages at the command line to `pipenv`, the `--dev` flag does indeed add those packages to the develop packages.
